### PR TITLE
Nazanin's news site

### PR DESCRIPTION
**Pull request by nazi-pira**

### CODE REVIEW

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
To meet the requirements:
- Publish your site on Netlify and add a link to it to the readme
- Style your page. - You have padding/margin(white frame) around your content 
- Create a responsive design with the following views; Desktop 4 columns, tablet 2 columns, mobile 1 column.
- The site should have a header, the main news article, a grid of news articles.

**Comment by nazi-pira** [comment]
Fixed

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,148 @@
-html {
```
Comment (KarolCislowski) [inline code/style.css:1]: You should remember about default margin/padding of your body and HTML. the best way to reset all sneaky box behaviors you can use 
```css
* {
    margin: 0;
    padding: 0;
  }
```

**Review by KarolCislowski — APPROVED** [review]
Nice 😄

**Pull request closed without merging**